templates/default/custom.properties.erb

```diff
@@ -388,3 +388,6 @@ edu.harvard.dce.production.email=<%= @production_management_email %>
 edu.harvard.dce.cas.login.url=https://www.pin1.harvard.edu/cas/login
 edu.harvard.dce.cas.service=<%= @cas_service %>
 
+# Immersive classroom base url. Used to push a notification when a video is replaced by its trimmed version.
+edu.harvard.dce.immersive.classroom.url=<%= @immersive_classroom_url %>
```

Comment:
Will it cause problems for this to get the string 'None' when `immersive_classroom_url` isn't set?

Reply:
Hi Jay, good catch. This param is part of Rute's notify IC feature. Her OC feature code snippet below, shows that this is expecting a string or empty string. So, I'll change the default to this from 'None' to '' empty string.

```
+    // Skip if immersive classroom url was not set. This is OK for a dev cluster.
+    if (StringUtils.isEmpty(immersiveBaseUrl)) {
+      return createResult(Action.SKIP);
+    }
```

Reply:
Changed to "" so the OC code will skip default param when not set in a cluster config.